Added space between text and link in Get Started page

ShenpaiSharma opened the pull request:
Fixes #926

 Changes: 
There was no space between text and link, which was making it difficult to read.

 Screenshots of the change: 
Previously:
![image](https://user-images.githubusercontent.com/47415702/105179458-62057180-5b4f-11eb-9a52-0a5b19dbaf2a.png)

After Fix:
![image](https://user-images.githubusercontent.com/47415702/105179532-76496e80-5b4f-11eb-8e7a-6768f16106ce.png)

limzykenneth commented:
The space shouldn't be added in the templates themselves as it would be problematic when it comes to translating this page, not all languages uses spaces to delineate individual words. The space should be added in the language data files for English and possibly Spanish instead.

ShenpaiSharma commented:
Hi @limzykenneth, I removed that space from templates, and instead, I added that space in English translation only as Spanish translation seems all fine to me.

limzykenneth commented on src/data/en.yml:
```diff
@@ -159,7 +159,7 @@ get started:
     the p5 online editor, outside the editor you must add the accessibility
     library in your html. To learn more visit
-  your-first-sketch7: using p5 with a screen reader tutorial
+  your-first-sketch7: ' using p5 with a screen reader tutorial'
```
Shouldn't the space be added to the end of the previous entry before the link instead of being a part of the link itself?

ShenpaiSharma replied:
Yeah, but in another translation as in Spanish, it was added in the link part only.

limzykenneth replied:
No need to match that in this case as the space fits better not as part of this link here.

ShenpaiSharma replied:
Ok, then I will add at the end of the previous entry.

limzykenneth commented on src/data/en.yml:
```diff
@@ -158,8 +158,8 @@ get started:
     the p5 online editor, outside the editor you must add the accessibility
-    library in your html. To learn more visit
-  your-first-sketch7: ' using p5 with a screen reader tutorial'
+    library in your html. To learn more visit 
```
One last thing, the space here will probably get deleted at some point by virtue of being at the end of the line (my text editor for example is setup to trim whitespace at end of line), to prevent that we can enclose the whole string in quotes like you have done the line below.

ShenpaiSharma replied:
Hi @limzykenneth, I don't think it is possible to enclose the text part in quotes here because of the block chomping indicator ">-" used here, and if we even try to use the quotes here, then it's the sole purpose of adding space at every line break will be lost. To resolve the above-mentioned problem, we probably should add space in the link part only.

limzykenneth replied:
Hmm...would using a HTML entity code here work? (`&#32;` for space for example)

If not then I guess we'll have to revert to adding the space before the link.

ShenpaiSharma replied:
Yeah, sure, it will work here. Thank you so much...

limzykenneth reviewed (changes requested):
Thanks for being patient with these reviews here, just one more thing and we can merge

limzykenneth commented:
Perfect, thanks!
